### PR TITLE
Resolves #2010

### LINKED ISSUE #2010: [Bug]: No idle kick not working
### Describe the bug

I get kicked out of the lobby even if this feature is active

### Steps To Reproduce

Go to Self tab

Activate no idle kick

you will be kicked out when you stand idle

### Expected Behavior

You shouldn't be thrown when you're idle

### Logs and/or screenshots

[yimidlebug.txt](https://github.com/YimMenu/YimMenu/files/12401450/yimidlebug.txt)

### Store

Epic Games

### Additional context

_No response_

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

### PR DESCRIPTION
**Pull request by gir489returns**

### CODE REVIEW

File: src/backend/looped/system/self_globals.cpp [hunk 1]
```diff
@@ -20,6 +20,17 @@ namespace big
...
+		if (original_heap_size != 650)
+		{
+			LOG(INFO) << "Original Memory Heap Size: " << original_heap_size << "MB";
+			DWORD old_heap_protect;
+			VirtualProtect(g_pointers->m_gta.m_heap_size, 4, PAGE_READWRITE, &old_heap_protect);
+			*g_pointers->m_gta.m_heap_size = 650 * 1024 * 1024;
+			VirtualProtect(g_pointers->m_gta.m_heap_size, 4, old_heap_protect, &old_heap_protect);
+			LOG(INFO) << "Modified Memory Heap Size To: 650MB";
+		}
+
```
Comment (Yimura) [inline src/backend/looped/system/self_globals.cpp]: Isn't this only helpful when injecting VERY early.

**Comment by D-Borkan** [comment]
Tested teleport and it works well now

File: src/backend/looped/system/self_globals.cpp (hunk 1 above)
Comment (gir489returns) [reply src/backend/looped/system/self_globals.cpp:33]: Should we remove it?

Comment (xiaoxiao921) [reply src/backend/looped/system/self_globals.cpp]: Yep this piece of code only works when you inject quite early during the game initialization, can be removed for now and added in a proper place later

**Comment by gir489returns** [comment]
@Yimura I think this PR is ready for merging. The idle issue will have to remain, I suppose. I don't know how to fix it as I can't replicate it.

**Review by Yimura — APPROVED** [review]

**Review by Yimura — APPROVED** [review]

**Pull request merged**